HomeWorkApp/src/main/java/iprody/coffeeshot/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,51 @@
+package iprody.coffeeshot;
+
+import java.util.*;
+
+public class CoffeeOrderBoard {
+    private final List<Order> orders; // Список текущих заказов
+    private int nextOrderNumber; // Следующий номер заказа
+
+    public CoffeeOrderBoard() {
+        this.orders = new ArrayList<>();
+        this.nextOrderNumber = 1; // Начинаем с 1
+    }
+
+    // Метод добавления нового заказа
+    public void add(String name) {
+        Order newOrder = new Order(nextOrderNumber++, name);
+        orders.add(newOrder);
+    }
+
+    // Метод выдачи ближайшего заказа (первый в очереди)
+    public Order deliver() {
```

Comment:
Нул возвращать плохо, как обсуждали на разборе.
```suggestion
    public Optional<Order> deliver() {
```